docs/source/export.rst

```diff
@@ -0,0 +1,113 @@
+Exporting NWB files
+===================
+
+Export is a new feature in PyNWB 2.0 (adapted from HDMF 2.0). You can use export to take an
```

Comment:
I'd rather not say this is "new". Are we going to edit this document in a year when the feature is no longer new?